index.html

```diff
@@ -20,7 +17,17 @@
         </ul>
       </nav>
     </header>
-    <main></main>
+    <main>
+      <section id="search-bar"></section>
+        <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+        <p>En plein centre ville ou en pleine nature</p>
+        <form method="post" action="#">
+            <input type="text" name="search" id="search" placeholder="Marseille, France"/>
+            <label for="search"></label>
+        <button>Rechercher</button>
```

Comment:
add type submit and put text inside a span + add a comment for icon like <!-- icon -->